### DISCUSSION TITLE
Jet mode activation

### DISCUSSION THREAD

**Discussion by mptr8956** [JanM321/esphome-lg-controller#46] [Q&A] [answered]
Hi, my unit supports Jet mode in cooling but I haven't found a switch or button for it in home assistant. Do I need to build it myself and if so, how do I do it?

**Comment by JanM321** [comment, marked as answer]
There's some discussion on Jet Mode in #38.

I'm planning to look into adding this in a few months in the cooling season. You could try adding a switch for it yourself, but it might not be super easy.

**Comment by mptr8956** [comment]
thanks, i guess thats above my coding skills. i'll wait for your implementation then...shouldnt bee too long until the cooling season starts :D
